docs/basics.adoc

```diff
@@ -1476,6 +1492,17 @@ By default, JanusGraph uses the Astyanax library to connect to Cassandra cluster
 
 When numerous clients are connecting to Elasticsearch, it is likely that an `OutOfMemoryException` occurs. This is not due to a memory issue, but to the OS not allowing more threads to be spawned by the user (the user running Elasticsearch). To circumvent this issue, increase the number of allowed processes to the user running Elasticsearch. For example, increase the `ulimit -u` from the default 1024 to 10024.
 
+=== Dropping a Database
+
+TO drop a database using the Gremlin Console you can call org.janusgraph.core.util.JanusGraphCleanup.clear(graph). The graph you want to drop needs to be defined and closed prior to running JanusGraphCleanup.clear().
```

Comment:
s/TO/To

add backticks around the code pieces in this paragraph.

`JanusGraphCleanup.clear(graph)` is [deprecated](https://github.com/JanusGraph/janusgraph/blob/v0.2.0/janusgraph-core/src/main/java/org/janusgraph/core/util/JanusGraphCleanup.java#L46). `JanusGraphFactory.drop(graph)` should be used instead.

Reply:
added benefit is that `JanusGraphFactory` is already on the import list for Gremlin Console, so you wouldn't need to include the full-qualified package name.